docs: clarify BigQuery WIF is not supported in dbt Fusion (#9034)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-bigquery-oauth.md

## Set up BigQuery Workload Identity Federation <Lifecycle status= "managed" /> 

> :::note dbt Fusion

Author: ```suggestion
:::info WIF not supported in <Constant name="fusion"/>
```

> Workload Identity Federation (WIF) is not currently supported in dbt Fusion. If you're using dbt Fusion, use a Service Account or Native OAuth connection instead. Support is coming soon.

Author: ```suggestion
Workload Identity Federation (WIF) is not currently supported in <Constant name="fusion"/>. If you're using <Constant name="fusion"/>, use a Service Account or Native OAuth connection instead. Support is coming soon.
```

Workload Identity Federation (WIF) allows application workloads, running outside the <Constant name="dbt_platform" />, to act as a service account without the need to manage service accounts or other keys for deployment environments. The following instructions will enable you to authenticate your BigQuery connection in the <Constant name="dbt_platform" /> using WIF. 
Currently, Microsoft Entra ID is the only supported identity provider (IdP). If you need additional IdP support, please contact your account team.

:::info WIF not supported in <Constant name="fusion_engine"/>

> WIF is not currently supported in <Constant name="fusion"/>. If you're using the <Constant name="fusion_engine"/>, use Native OAuth connection instead. Support for WIF is coming soon.

Author: ```suggestion
WIF is not currently supported in [<Constant name="fusion"/](/docs/fusion/supported-features?version=2.0#bigquery)>. If you're using the <Constant name="fusion_engine"/>, use Native OAuth connection instead. Support for WIF is coming soon.
```

> WIF is not currently supported in <Constant name="fusion"/>. If you're using the <Constant name="fusion_engine"/>, use a Service Account or Native OAuth connection instead. Support is coming soon.

Author: ```suggestion
WIF is not currently supported in <Constant name="fusion"/>. If you're using the <Constant name="fusion_engine"/>, use Native OAuth connection instead. Support for WIF is coming soon.
```

> WIF is not currently supported in [<Constant name="fusion"/](/docs/fusion/supported-features?version=2.0#bigquery)>. If you're using the <Constant name="fusion_engine"/>, use Native OAuth connection instead. Support for WIF is coming soon.

Reviewer: Link fix + a suggestion to make it more concise
```suggestion
WIF is not currently supported in [<Constant name="fusion"/>](/docs/fusion/supported-features?version=2.0#bigquery). Use a [Native OAuth connection](#set-up-bigquery-native-oauth) or service account instead. Support for WIF is coming soon.
```

> WIF is not currently supported in [<Constant name="fusion"/>](/docs/fusion/supported-features?version=2.0#bigquery). Use a [Native OAuth connection](#set-up-bigquery-native-oauth) or service account instead. Support for WIF is coming soon.

Author: ```suggestion
WIF is not currently supported in [<Constant name="fusion"/>](/docs/fusion/supported-features?version=2.0#bigquery). Use a [Native OAuth connection](#set-up-bigquery-native-oauth) or service account instead. [Support for WIF](https://github.com/dbt-labs/dbt-fusion/issues/1629) is coming soon.
```
